chart/k8gb/templates/role.yaml

```diff
@@ -7,54 +7,11 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - pods
-  - nodes
   - services
-  - services/finalizers
   - endpoints
```

Comment:
For Endpoints we probably need only `get` verb

Reply:
makes sense